tests/integration/llm/prepare.py

```diff
@@ -870,7 +870,8 @@
         "option.max_rolling_batch_size": 16,
         "option.tokenizer_mode": "mistral",
         "option.limit_mm_per_prompt": "image=4",
-        "option.entryPoint": "djl_python.huggingface"
+        "option.entryPoint": "djl_python.huggingface",
+        "option.tensor_parallel_degree": "max"
```

Comment:
QQ: Is tp_degree needed here? Since the default should be max?

Reply:
i should maybe add a comment here. For this specific case it is needed. This test is using vllm, so not MPI mode meaning it doesn't get the default tp max logic. It's also not a typical huggingface model so it doesn't go through the auto configuration path which would set tp degree.